docs/src/index.md

```diff
@@ -49,6 +49,26 @@ LibPQ.load!(
     "INSERT INTO libpqjl_test (no_nulls, yes_nulls) VALUES (\$1, \$2);",
 )
 
+using DataFrames
```

Comment:
This addition should actually replace the code under the `COPY` header.

Reply:
done